nodes.go

```diff
@@ -576,11 +576,20 @@ func setField(tokens []lexer.Token, strct reflect.Value, field structLexerField,
 	}
 
 	if f.Kind() == reflect.Slice {
-		fieldValue, err = conform(f.Type().Elem(), fieldValue)
-		if err != nil {
-			return err
+		if d, ok := reflect.New(f.Type().Elem()).Interface().(Capture); ok {
+			for _, v := range fieldValue {
+				if err := d.Capture([]string{v.Interface().(string)}); err != nil {
```

Comment:
I don't think this code is correct. You should be constructing a new `d` for each element appended to `f`. I think this is only working in the tests at the moment because each `d` is a value and gets copied. If `d` were a pointer to a struct, for example, it would get repeatedly overwritten.

Reply:
Indeed, it should be fixed now.